test/distributed/test_c10d.py

```diff
@@ -24,6 +24,7 @@
 import torch.nn.functional as F
 import torch.distributed as c10d
 import torch.distributed as dist
+from torch._C._distributed_c10d import TCPStore
```

Comment:
hmm, does this mean applications should no longer access `torch.distributed.TCPStore`?

Reply:
Yes, init TCPStore from torch.distributed will show no attribute error. 
AttributeError: module 'torch.distributed' has no attribute 'TCPStore'

Reply:
This will be a BC issue right? We should still support importing TCPStore from torch.distributed, given that most users will access it as such:

```
import torch.distributed as dist
store = dist.TCPStore(...)
```

Reply:
Yes, I think we should add it back, since it will break users code which use TCPStore from torch.distributed.